go/erasablekv/erasable_kv_store.go

```diff
@@ -44,7 +44,12 @@ const noiseSuffix = ".ns"
 const storageSubDir = "eraseablekvstore"
 
 func getStorageDir(g *libkb.GlobalContext, subDir string) string {
-	return filepath.Join(g.Env.GetDataDir(), storageSubDir, subDir)
+	base := g.Env.GetDataDir()
+	// check for iOS
+	if runtime.GOOS == "darwin" && g.GetAppType() == libkb.MobileAppType {
```

Comment:
@joshblum this is a little janky, but this is what I came up with to make it so the EKs come along for the migration ride on iOS. `DataDir` == `ConfigDir` except on XDG platforms.